ipaserver/install/server/install.py

```diff
@@ -940,9 +940,24 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
     print("")
-    print("Next steps:")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("\nObtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
+    print("\nServer is installed with following roles :")
+    print("* Directory Server")
```

Comment:
IMO ``LDAP Server`` is better because it's self-explaining.

Reply:
Maybe `389 Directory Server (LDAP Server)`?